cmd/lncli/commands.go

```diff
@@ -2267,6 +2279,11 @@ func payInvoice(ctx *cli.Context) error {
 		FeeLimit:       feeLimit,
 		OutgoingChanId: ctx.Uint64("outgoing_chan_id"),
 	}
+
+	if ctx.IsSet(cltvLimitFlag.Name) {
+		req.CltvLimit = uint32(ctx.Int(cltvLimitFlag.Name))
```

Comment:
won't always assigning this value just default it to 0 if not present?

Reply:
Yes you are right. Simplified.